docs/visual-testing/integrations/java.md

```diff
@@ -331,7 +331,17 @@ visual.sauceVisualCheck("Inventory Page", options);
 
 ### Full page screenshots
 
-<FullPageDescription />
+By default, only the current viewport is captured when `.sauceVisualCheck` is used. You can opt in to capturing the entire page by using the `enableFullPageScreenshots` option. It will capture everything by scrolling and stitching multiple screenshots together.
+Additionally, you have the option to configure full page settings using the `FullPageConfig` option.
+
+:::note
+It's recommended to use the `HideAfterFirstScroll` option for fixed or sticky position elements such as sticky headers or consent banners.
```

Comment:
Is this correct for java ? Isn't it expected to use Builder ?

Casing looks off too.

Reply:
fixed